VPLAY-10178 : application getting crashed due to async task thread leak

Gnanesha opened the pull request:
Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431

Commit:
VPLAY-10178 : application getting crashed due to async task thread leak

Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431
Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

Commit:
VPLAY-10178 : application getting crashed due to async task thread leak

Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431
Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=232) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=232) before we can accept your contribution.<br/><hr/>**Gnanesha** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=232) it.</sub>

Commit:
VPLAY-10178 : application getting crashed due to async task thread leak

Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431
